authnz/src/main/java/io/stargate/auth/AuthnzService.java

```diff
@@ -15,11 +15,30 @@
  */
 package io.stargate.auth;
 
-public interface AuthenticationService {
+import io.stargate.db.datastore.ResultSet;
+import io.stargate.db.schema.Table;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public interface AuthnzService {
```

Comment:
WDYT about adding a separate interface for the "authz" part (even if we implement both in the same class)?

Reply:
I thought about having an Authentication.java and Authorization.java but wasn't sure if there would be enough methods to warrant having both.

Reply:
In my head, the nature of those "execute with permission" methods is very different from create/validate token, hence my wish to keep them in different classes / interfaces... just my 2 cents.